fenix/app/src/main/java/org/mozilla/fenix/settings/wallpaper/WallpaperSettings.kt

```diff
@@ -300,7 +300,7 @@ private fun WallpaperThumbnailItem(
             .fillMaxWidth()
             .aspectRatio(aspectRatio)
             .then(border)
-            .clickable { onSelect(wallpaper) }
+            .debouncedClickable(onClick = { onSelect(wallpaper) })
```

Comment:
```suggestion
            .debouncedClickable {
                onSelect(wallpaper) 
            }
```

I think this can be simplified slightly.